soaks/tests/http_datadog_filter_blackhole/terraform/vector.toml

```diff
@@ -16,228 +18,16 @@ encoding = "ndjson"
-
-[transforms.azure_web]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:azure.web"
-
-[transforms.azure_storage]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:azure.storage"
-
-[transforms.azure_network]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:azure.network"
-
-[transforms.azure_compute]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:azure.compute"
-
-[transforms.etcd]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:etcd"
-
-[transforms.glog_pipeline]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:(admission-webhook OR api-server OR cert-manager-acmesolver OR cert-manager-cainjector OR cert-manager-controller OR cert-manager-webhook OR cluster-proportional-autoscaler-amd64 OR hyperkube OR ip-masq-agent OR k8s-prometheus-adapter-amd64 OR kube-apiserver OR kube-controller-manager OR kube-proxy OR kube-state-metrics OR metacontroller OR metrics-server-amd64 OR prometheus-operator OR vpa-admission-controller OR vpa-recommender OR vpa-updater)"
-
-[transforms.auth0]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:auth0"
-
-[transforms.kube_scheduler__glog_]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:(kube_scheduler OR kube-scheduler)"
-
-[transforms.aws_ecs_agent]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:amazon-ecs-agent"
-
-[transforms.nodejs]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:nodejs"
-
-[transforms.postgresql]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:postgresql"
-
-[transforms.cassandra]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:cassandra"
-
-[transforms.apache_httpd]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:httpd"
-
-[transforms.azure_recovery_services]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:azure.recoveryservices"
-
-[transforms.c_]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:csharp"
-
-[transforms.web_browser_logs]
-type = "filter"
-inputs = ["logs"]
-condition.type = "datadog_search"
-condition.source = "source:browser"
-
-##
-## Sinks
-##
+.custom.message = .message
+.tags = []
+'''
```

Comment:
Should this be here?

Reply:
Yeah, I missed that when deleting the preprocessing. [194e890](https://github.com/vectordotdev/vector/pull/11407/commits/194e890f1d24aeb09100b7103cdb1116d644c837) fixes that and another config error.